lib/gitlab/ci/templates/jobs/gradle/DependencyScanning.yml

```diff
@@ -1,4 +1,14 @@
+include:
+  - template: Security/Dependency-Scanning.gitlab-ci.yml
+  - remote: https://raw.githubusercontent.com/chesapeaketechnology/gitlab-templates/get-custom-gradle-working-and-move-chmod-before-script/lib/gitlab/ci/templates/references/gradle/GradleJavaBase.yml
```

Comment:
Move this from get-custom-gradle-working-and-move-chmod-before-script to release/4 before merging.